graph-cli npm package version 0.4.0 cannot be installed.

I tried myself.
```
yarn add @graphprotocol/graph-cli
```
works fine but
```
npm install @graphprotocol/graph-cli
```
fails with the following error:
```
➜  foo$ npm install @graphprotocol/graph-cli
npm WARN enoent ENOENT: no such file or directory, open '/private/tmp/foo/package.json'
npm WARN foo No description
npm WARN foo No repository field.
npm WARN foo No README data
npm WARN foo No license field.

npm ERR! path /private/tmp/foo/node_modules/@graphprotocol/graph-cli/graph-build.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/private/tmp/foo/node_modules/@graphprotocol/graph-cli/graph-build.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jannis/.npm/_logs/2018-11-04T18_32_59_695Z-debug.log
```

Yap.npm install fail

On Sun, 4 Nov 2018 at 20:38 Jannis Pohlmann <notifications@github.com>
wrote:

> I tried myself.
>
> yarn add @graphprotocol/graph-cli
>
> works fine but
>
> npm install @graphprotocol/graph-cli
>
> fails with the following error:
>
> ➜  foo$ npm install @graphprotocol/graph-cli
> npm WARN enoent ENOENT: no such file or directory, open '/private/tmp/foo/package.json'
> npm WARN foo No description
> npm WARN foo No repository field.
> npm WARN foo No README data
> npm WARN foo No license field.
>
> npm ERR! path /private/tmp/foo/node_modules/@graphprotocol/graph-cli/graph-build.js
> npm ERR! code ENOENT
> npm ERR! errno -2
> npm ERR! syscall chmod
> npm ERR! enoent ENOENT: no such file or directory, chmod '/private/tmp/foo/node_modules/@graphprotocol/graph-cli/graph-build.js'
> npm ERR! enoent This is related to npm not being able to find a file.
> npm ERR! enoent
>
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/jannis/.npm/_logs/2018-11-04T18_32_59_695Z-debug.log
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/graphprotocol/graph-cli/issues/164#issuecomment-435694542>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFt3ngrM1tUbM-Rzje36X8L5PR0CWV-Oks5urzQ1gaJpZM4YNXZ_>
> .
>

--- a/package.json
+++ b/package.json
@@ -21,9 +21,7 @@
     "winston": "^3.0.0"
   },
   "bin": {
-    "graph": "./graph.js",
-    "graph-build": "./graph-build.js",
-    "graph-codegen": "./graph-codegen.js"
+    "graph": "./graph.js"
   },
   "devDependencies": {
     "jest": "^23.6.0"

